src/main/java/core/org/akaza/openclinica/bean/extract/odm/ClinicalDataReportBean.java

```diff
@@ -215,10 +215,22 @@ public void addNodeClinicalData(boolean header, boolean footer) {
                             xml.append("\" OpenClinica:EndDate=\"" + StringEscapeUtils.escapeXml(endDate));
                         }
 
+                        if (BooleanUtils.isTrue(se.getRemoved()))
```

Comment:
You could probably clean this up even more by adding utility/transient methods in StudyEvent and EventCrf classes that use BooleanUtils and call those methods directly instead of using this BooleanUtils logic everywhere.
E.g. 
public boolean isRemoved() {
  return BooleanUtils.isTrue(this.getRemoved())
}

and now you can just call "studyEvent.isRemoved()" method

Reply:
All done, create transient methods in domain objects